assets/sass/woocommerce/_variables.scss

```diff
@@ -0,0 +1,21 @@
+/**
+ * WooCommerce CSS Variables
+ */
+
+$woocommerce:   	#a46497;
+$green:         	#7ad03a;
+$red:           	#a00;
+$orange:        	#ffba00;
+$blue:          	#2ea2cc;
```

Comment:
-1
Não misture espaços com tabs. Tabs no início da linha, espaço para fazer o alinhamento.